images/base/fs/etc/snopsrepo.d/base.json

```diff
@@ -45,7 +45,7 @@
 		{
 			"name":"f5-application-services-integration-iApp",
 			"repo":"https://github.com/F5Networks/f5-application-services-integration-iApp.git",
-			"branch":"develop",
+			"branch":"master",
```

Comment:
Unbreaking the build for now.  Going to move to release tags/commit hashes soon.  The commit of the new build framework to develop broke a lab at Agility because we weren't being being strict enough with our branch/commit refs.